src/Compilers/Core/Portable/Symbols/ILocalSymbol.cs

```diff
@@ -23,11 +23,17 @@ public interface ILocalSymbol : ISymbol
         bool IsConst { get; }
 
         /// <summary>
-        /// Returns true if this local is a ref-local.
+        /// Returns true if this local is a ref local or a ref readonly local.
+        /// Use <see cref="RefKind"/> to get more detailed information.
         /// </summary>
```

Comment:
So, "IsRef" is true for readonly references as well. 
I do not have a strong opinion on this, but we have similar API for methods/properties, should we make sure it is also "inclusive"?

Reply:
> So, "IsRef" is true for readonly references as well.

Yes.

> I do not have a strong opinion on this, but we have similar API for methods/properties, should we make sure it is also "inclusive"?

IMethodSymbol and IPropertySymbol also have ReturnsByRefReadonly property, I'll let you decide what correlation should be between it and ReturnsByRef property. If any adjustments are needed, I think they are out of scope of this PR.